base/markdown/render/latex.jl

```diff
@@ -69,6 +69,8 @@ function latex(io::IO, md::List)
     end
 end
 
+latex(io::IO, md::LaTeX) = print(io, md.formula)
+
 function writemime(io::IO, ::MIME"text/latex", md::HorizontalRule)
```

Comment:
Off Topic: but this should probably also be the latex function rather than writemime. (I think that was my fault).